package.json

```diff
@@ -29,6 +29,9 @@
     "watch-electron": "onchange -i -k 'build/**/*' 'backend/build/**/*' -- npm run electron"
   },
   "dependencies": {
+    "@date-io/date-fns": "^1.3.13",
+    "@material-ui/pickers": "^3.2.10",
+    "date-fns": "^2.16.1",
```

Comment:
I don't believe these are needed in the root level package.json for electron, are they?